Makefile

```
@@ -66,8 +67,26 @@ docker-version-check:
 	fi; \
 	if [ ! $$(docker buildx version | cut -d' ' -f 2) ]; then \
 		echo "Docker buildx does not exist can't continue"; \
-	fi;
+		exit 1;\
+	fi; \
+	echo "Docker version: $${DOCKER_VERSION}."; \
+	echo "Docker Compose version: $${DOCKER_COMPOSE_VERSION}.";
```

```suggestion
	echo "Docker Compose version: $${DOCKER_COMPOSE_VERSION}."
```
The last line shouldn't need a semicolon

---

fixed in 8deb24eb